test/search/test_buckets.py

```diff
@@ -29,7 +29,6 @@ def setUp(self):
                         {"value": "record", "count": 37470380},
                         {"value": "tna", "count": 26008838},
                         {"value": "nonTna", "count": 16454377},
-                        {"value": "digitised", "count": 9055592},
```

Comment:
`{"value": "digitised", "count": 9055592},` should be retained. It will ensure the buckets for display do not consider Online records.

Reply:
Thanks @TNA-Allan - I've updated the commit and pushed.